chore: remove tee override (#2136)
berops/claudie: docs/input-manifest/gpu-example.md

```bash
helm install --wait --generate-name \
    -n gpu-operator --create-namespace \
    nvidia/gpu-operator --set cdi.enabled=true --set cdi.nriPluginEnabled=true
```

> !!! note "Claudie overrides `/etc/containerd/config.toml` on every reconciliation, thus to avoid conflicts with the overrides the [`cdi` with `nri`](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/cdi.html) plugin are enabled that bypass the need for a consistent `/etc/containerd/config.toml` for the operator"

Reviewer: ```suggestion
!!! note "Claudie overrides `/etc/containerd/config.toml` on every reconciliation loop. To avoid conflicts with these overrides the [`cdi` and `nri`](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/cdi.html) plugins are enabled. This bypasses the conflict with Claudie-reconciled `/etc/containerd/config.toml` for the operator."
```
